<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../icons/logo.png</normaloff>../../icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget {
	border-image: url(icons/homeBG.png) 0 0 0 0 stretch stretch;
}</string>
    </property>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/exitButton.png</normaloff>../../icons/exitButton.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="logoLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="employeeButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#employeeButton {
	background-color: transparent;
	background-image: url(icons/employeeButton.png);
	border-image: url(icons/employeeButton.png) 0 0 0 0 stretch stretch;
}

#employeeButton:Hover {
	background-color: transparent;
	background-image: url(icons/employeeButtonHover.png);
	border-image: url(icons/employeeButtonHover.png) 0 0 0 0 stretch stretch;
}

#employeeButton:pressed {
	background-color: transparent;
	background-image: url(icons/employeeButtonClicked.png);
	border-image: url(icons/employeeButtonClicked.png) 0 0 0 0 stretch stretch;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Display</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #3169B0;</string>
     </property>
     <property name="text">
      <string>Select Your Role</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adminButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#adminButton {
	background-color: transparent;
	background-image: url(icons/adminButton.png);
	border-image: url(icons/adminButton.png) 0 0 0 0 stretch stretch;
}

#adminButton:Hover {
	background-color: transparent;
	background-image: url(icons/adminButtonHover.png);
	border-image: url(icons/adminButtonHover.png) 0 0 0 0 stretch stretch;
}

#adminButton:pressed {
	background-color: transparent;
	background-image: url(icons/adminButtonClicked.png);
	border-image: url(icons/adminButtonClicked.png) 0 0 0 0 stretch stretch;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
